src/ImageSharp/Advanced/ParallelRowIterator.cs

```diff
@@ -65,8 +65,8 @@ public static void IterateRows<T>(
         }
 
         int verticalStep = DivideCeil(rectangle.Height, numOfSteps);
-        var parallelOptions = new ParallelOptions { MaxDegreeOfParallelism = numOfSteps };
-        var wrappingOperation = new RowOperationWrapper<T>(top, bottom, verticalStep, in operation);
+        ParallelOptions? parallelOptions = new() { MaxDegreeOfParallelism = numOfSteps };
```

Comment:
This shouldn't be nullable.

Reply:
Nor below, not sure how that builds.

Reply:
Oh damn rider. Wait a minute before you waste your time. gonna look over it first

Reply:
ok made a commit and hopefully fixed all places